src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -865,50 +968,70 @@ private async Task<Dictionary<KeyValueIdentifier, ConfigurationSetting>> LoadKey
             return watchedSettings;
         }
 
-        private Dictionary<KeyValueIdentifier, ConfigurationSetting> UpdateWatchedKeyValueCollections(Dictionary<KeyValueIdentifier, ConfigurationSetting> watchedSettings, IDictionary<string, ConfigurationSetting> existingSettings)
+        private async Task<List<KeyValueChange>> GetRefreshedKeyValueCollections(
+            IEnumerable<KeyValueWatcher> featureFlagWatchers,
```

Comment:
This parameter name is incorrect. This method is used for both selected key value and feature flags.